metabase/metabase
# Define the `PublicSettings` type

This PR continues to break down the ambiguous `Settings` type into smaller, well-defined settings based on their nature. We're defining all Public settings here.

> [!Note]
> All these settings should be available for the unauthenticated user.

### How to verify?
1. Run a local Metabase instance.
2. Clear the network tab
3. Sign out
4. Check the `GET /api/session/properties` response

Every single key listed in the `PublicSettings` type should be there.
Keep in mind that not all possible Metabase public settings have been documented/defined on the Frontend in the first place so you shouldn't be surprised if you don't see them in this type.
